connections/simopen.md

```diff
@@ -0,0 +1,72 @@
+# Simultaneous Open for multistream-select
```

Comment:
This spec only applies to connection bootstrapping, whereas multistream-select is used in other contexts too (stream-level protocol negotiation). So I'd title this differently, e.g.:

```
# Connection bootstrapping: handling TCP simultaneous open in multistream-select.
```

Reply:
Sure, we can change the title. Go ahead and pick something you like!

Reply:
Also, this works for UDP too!